src/test/java/com/auth0/json/auth/TokenHolderTest.java

```diff
@@ -3,25 +3,75 @@
 import com.auth0.json.JsonTest;
 import org.junit.Test;
 
+import java.time.Instant;
+import java.util.Date;
+
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.*;
 
 public class TokenHolderTest extends JsonTest<TokenHolder> {
 
-    private static final String json = "{\"id_token\":\"eyJ0eXAiOiJKV1Qi...\",\"access_token\":\"A9CvPwFojaBI...\",\"refresh_token\":\"GEbRxBN...edjnXbL\",\"token_type\": \"bearer\",\"expires_in\":86000, \"scope\": \"openid profile email\"}";
-
     @Test
     public void shouldDeserialize() throws Exception {
+        String json = "{\"id_token\":\"eyJ0eXAiOiJKV1Qi...\",\"access_token\":\"A9CvPwFojaBI...\",\"refresh_token\":\"GEbRxBN...edjnXbL\",\"token_type\": \"bearer\",\"expires_in\":86000, \"scope\": \"openid profile email\"}";
+
+        TokenHolder holder = fromJSON(json, TokenHolder.class);
+
+        assertThat(holder, is(notNullValue()));
+        assertThat(holder.getAccessToken(), is("A9CvPwFojaBI..."));
+        assertThat(holder.getIdToken(), is("eyJ0eXAiOiJKV1Qi..."));
+        assertThat(holder.getRefreshToken(), is("GEbRxBN...edjnXbL"));
+        assertThat(holder.getTokenType(), is("bearer"));
+        assertThat(holder.getExpiresIn(), is(86000L));
+        assertThat(holder.getScope(), is("openid profile email"));
+
+        // allow for a small tolerance since now may be calculated at slightly different times in the test
+        assertThat(getExpectedExpiredAtDiff(86000, holder.getExpiresAt()), is(lessThanOrEqualTo(2L)));
```

Comment:
Provided a tolerance for the value, rather than refactoring to inject a mock clock, as that seemed like overkill)